src/main/flashwavrecorder/RecorderJSInterface.as

```diff
@@ -250,6 +251,16 @@ package flashwavrecorder {
       return MultiPartFormUtil.base64_encdode(data);
     }
 
+    private function getWAV(name:String):ByteArray {
+      var data:ByteArray;
+      try {
+        data = _recorder.convertToWav(name);
+      } catch (e:Error) {
+        data = new ByteArray();
+      }
+      return data;
```

Comment:
Unfortunately that will not work. It seem that there is no way to send binary data between flash and javascript. All data need to be converted to string. I found some questions on SO on this topic:
http://stackoverflow.com/q/22952032/1346003
http://stackoverflow.com/a/11811097/1346003